src/components/Field.jsx

```diff
@@ -37,17 +39,24 @@ const Field = (props) => {
 export default Field
 
 export const FieldWrapper = ({ required, label, dirty, touched, errors, children }) => {
-  var classes = classNames('account-field', {
```

Comment:
We need to map them in a styles[className]. I renamed the variable `classes` to indicate that we are first dealing with conditional classes.